c/Makefile

```diff
@@ -29,11 +29,13 @@
 # - CFLAGS: Any extra user-specified compiler flags (can be blank).
 
 # Recommended compiler flags:
-CFLAGS += -std=c99 -O
+CFLAGS ?= -std=c99 -O
```

Comment:
Don't remove -std flag. It needed during build. Correct row should be:
`CFLAGS += -std=c99`

Reply:
adjusted